Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/Events.proto

```diff
@@ -3,17 +3,29 @@
 
 syntax = "proto3";
 
+import "tools/google/protobuf/timestamp.proto";
+import "tools/google/protobuf/duration.proto";
+
 package BuildXL.Execution.Analyzer;
 
 option csharp_namespace = "BuildXL.Execution.Analyzer";
 
+//
+//  Generic Event Info (key, count, etc)
+//
+
+// Message that keeps track of total number of events processed
+message EventCount_XLGpp{

```

Comment:
>EventCount_XLGpp [](start = 8, length = 16)

why is it needed here, i.e., in proto file?

Reply:
Wanted to keep everything stored in the rocksDB as part of the proto file. Furthermore, this count will be useful for unit tests to determine that all the events have indeed been logged.

---
In reply to: [308889834](https://github.com/microsoft/BuildXL/pull/671#discussion_r308889834) [](ancestors = 308889834)